src/Compilers/CSharp/Portable/Syntax/CompilationUnitSyntax.cs

```diff
@@ -25,6 +25,16 @@ internal IList<ReferenceDirectiveTriviaSyntax> GetReferenceDirectives(Func<Refer
             return firstToken.GetDirectives<ReferenceDirectiveTriviaSyntax>(filter);
         }
 
+        /// <summary>
+        /// Returns #r directives specified in the compilation.
+        /// </summary>
+        public IList<LoadDirectiveTriviaSyntax> GetLoadDirectives()
```

Comment:
Should it also return erroneous directives later in the file (or in regular files)?